k8s/sandbox/cluster-00/jenkins-test/jenkins.yaml

```diff
@@ -319,16 +319,92 @@ spec:
                         shutdownOnIdle: false
                         storageAccountNameReferenceType: "existing"
                         storageAccountType: "Standard_LRS"
-                        subnetName: "jenkins-subnet"
+                        subnetName: "jenkins"
                         templateDesc: "Jenkins build agents for HMCTS"
                         templateDisabled: false
                         templateName: "cnp-jenkins-builders"
                         usageMode: "Use this node as much as possible"
                         usePrivateIP: true
                         virtualMachineSize: "Standard_D4_v3"
-                        virtualNetworkName: "mgmt-infra-sandbox"
-                        virtualNetworkResourceGroupName: "mgmt-infra-sandbox"
-
+                        virtualNetworkName: "core-sbox-vnet"
+                        virtualNetworkResourceGroupName: "core-infra-sbox-rg"
+                - azureVM:
+                    azureCredentialsId: "jenkinsServicePrincipal"
+                    cloudName: "cnp-azure-team-a"
+                    deploymentTimeout: 1200
+                    existingResourceGroupName: "mgmt-vmimg-store-sandbox"
+                    maxVirtualMachinesLimit: 40
+                    resourceGroupReferenceType: "existing"
+                    vmTemplates:
+                    - agentLaunchMethod: "SSH"
+                      agentWorkspace: "/opt/jenkins"
+                      builtInImage: "Windows Server 2016"
+                      credentialsId: "vm_agent_creds"
+                      diskType: "managed"
+                      doNotUseMachineIfInitFails: true
+                      enableMSI: false
+                      enableUAMI: true
+                      executeInitScriptAsRoot: true
+                      existingStorageAccountName: "mgmtvmimgstoresandbox"
```

Comment:
not sure about this either

Reply:
for now, I've created this `mgmtvmimgstoresandboxrpe` for test purpose